examples/shopify/shopify.js

```diff
@@ -0,0 +1,41 @@
+const { Reshuffle } = require('reshuffle')
+const { ShopifyConnector } = require('reshuffle-shopify-connector')
+
+// This example uses reshuffle-shopify-connector
+// Code and documentation available on Github: https://github.com/reshufflehq/reshuffle-shopify-connector
+
```

Comment:
Maybe add what the example does? e.g.
1. Lists up to 5 open orders
2. Lists all products
3. Creates a new country